app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params.dig(:session, :email)&.downcase
```

Comment:
cần `&.` chỗ downcase ko em, nil.downcase vẫn ổn mà

ngoài ra, suggest là việc load user và handle user not found em tạo hẵng 1 method để xử lý và dùng action filter là before action để auto call cho action create nè

ở method create khi vào tới đây thì sure là user khác nil rồi :D

Reply:
Em fix rồi ạ